pkg/kubelet/network/dns/dns.go

```diff
@@ -372,7 +377,11 @@ func (c *Configurer) GetPodDNS(pod *v1.Pod) (*runtimeapi.DNSConfig, error) {
 		// DNSNone should use empty DNS settings as the base.
 		dnsConfig = &runtimeapi.DNSConfig{}
 	case podDNSCluster:
-		clusterDNS := c.getClusterDNS(pod)
+		var clusterDNS []net.IP
+		if clusterDNS, err = c.getClusterDNS(pod); err != nil {
+			return nil, fmt.Errorf("failed to get DNS IP add: %v", err)
```

Comment:
add -> address

Reply:
thanks for pointing this typo out. will update.